integrations/dialpad/syncs/users.ts

```diff
@@ -0,0 +1,34 @@
+import type { NangoSync, ProxyConfiguration } from '../models';
+import type { DialpadUser, User } from '../types';
+
+export default async function fetchData(nango: NangoSync) {
+    
+    const proxyConfiguration: ProxyConfiguration = {
+ 
+        // https://developers.dialpad.com/reference/userslist
+        endpoint: '/users',
+        paginate: {
+            type: 'cursor',
+            cursor_name_in_request: 'cursor',
+            cursor_path_in_response: 'next_cursor',
```

Comment:
`Param limit_name_in_request is required for cursor pagination when limit is set`

So this should be added
```
limit_name_in_request: 'limit',
```